webpack/scenes/ContentViews/components/EnvironmentLabels.js

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import { Label } from '@patternfly/react-core';
 
 const EnvironmentLabels = (environments) => {
-  const { environments: singleEnvironment } = environments || {};
+  const { environments: singleEnvironment, isDisabled } = environments || {};
```

Comment:
Not sure if this was intended, but deconstructing props in this way (rather than one line above in the function declaration) is a sneaky way to avoid eslint requiring PropTypes. don't tell anyone :hand_over_mouth: